scipy/interpolate/interpolate.py

```diff
@@ -309,7 +309,7 @@ def _duplicate(ab):
     # transform to a pair (a, a)
     try:
         a, b = ab
-    except TypeError:
+    except (TypeError, ValueError):  # ndarray can rais ValueError here
```

Comment:
I can fix this typo and open the fix to `master` if people agree it's the right move